GETTING_STARTED.md

```diff
@@ -0,0 +1,127 @@
+# Getting Started
+
+While the [OCI Charter](./CHARTER.md) is our official governing document, many
+people have asked for a gentler introduction to the Open Container Initiative as
+well as general guidance on how to interact with the projects and specifications
+found under the OCI umbrella. This document attempts to be that starting point
+for those who may be new to the OCI, interested in participation, or who want to
+understand if a project is a fit for inclusion or contribution to the OCI.
+
+## What is the OCI?
+
+As our [website](https://opencontainers.org) states, chiefly the OCI is "an open
+governance structure for the express purpose of creating open industry standards
+around container formats and runtime." Created in 2015, the core initial purpose
+was to align Docker and other ecosystem players around a runtime and image
+specification that would become an industry baseline for how images are packaged
+and executed in OCI compliant runtimes.
+
+Since that initial work, finalized in 1.0 specs in the summer of 2017, a
+distribution specification is now underway, based on the HTTP API of the initial
+Docker distribution (registry) project. In 2019, an "artifacts" project was
+approved by the TOB and will be used to define other artifact types (in addition
+to OCI images) which may be defined and stored in registries conforming to the
+OCI distribution spec.
+
+## Who uses the OCI?
+
+It makes the most sense to see consumers of OCI as fitting into a few specific
+categories: contributors/members, implementors, and end users.
+
+**Contributors**, including the project maintainers, and member companies have a
+vested interest in bringing forward the "state of the art" with respect to the
+scope of the OCI. Currently this is of course limited to specifications around
+runtime, image, and distribution of containers. The artifacts repository and
+project is related to both image and distribution and is a natural expansion
+of OCI into ongoing experimentation with registries and cloud native tooling.
```

Comment:
The word experimentation is creating angst amongst some that see this as experimentation of registries that gives them the belief this will change. 
What I think you're trying to say is supporting other types, not as an experiment, but a commitment to support other types, enables others to experiment on a stable capability.

Reply:
Updated in latest commit.